src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Quadrangle {
+
+    public Square(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
+    }
```

Comment:
The `area` method in `Square` should not call `super.area()`, because `Quadrangle` might have a different way of calculating the area. `Square` should have its own implementation of the `area` method that calculates the area of a square based on its sides.